docs/poc2023q1/mailbox-controller.md

```diff
@@ -3,18 +3,20 @@
 The mailbox controller runs in the edge service provider workspace and
 maintains a child workspace per SyncTarget.
 
-## Temporary design detail
+## Linking SyncTarget with Mailbox Workspace
 
 For a given SyncTarget T, the mailbox controller currently chooses the
 name of the corresponding workspace to be the concatenation of the
 following.
 
 - the ID of the logical cluster containing T
 - the string "-w-"
```

Comment:
Didn't the above PR changes this to  `const wsNameSep = "-mb-"`

Reply:
Fixed oversight.